TrackingTools/GsfTools/plugins/BuildFile.xml

```diff
@@ -1,3 +1,4 @@
+<flags   CXXFLAGS="-Ofast"/>
```

Comment:
This is in contradiction with my test of the original version of this PR for 810